judo-requirement-report-test/src/test/java/hu/blackbelt/judo/requirement/test/TestRequirementProcessor.java

```diff
@@ -0,0 +1,238 @@
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test03",
+                        "OK",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test00",
+                        "There isn't any requirement id.",
+                        ""
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R04"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test02",
+                        "OK",
+                        "R02"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test01",
+                        "OK",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R05"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test05",
+                        "Missing annotation: @Test.",
+                        "R01"
+                }
+        );
+        refTable.add(
+                new String[]{
+                        "TestRequirementProcessor.test04",
+                        "OK",
+                        "R01"
+                }
+        );
+//        refTable.add(
+//                new String[]{
+//                        "",
+//                        "",
+//                        ""
+//                }
+//        );
+
+        return refTable
+                .stream()
+                .sorted(new RowComparator())
+                .collect(Collectors.toList());
+    }
+
+    private List<String[]> readCsv(String filePath) {
+        CSVParser parser = new CSVParserBuilder()
+                .withSeparator(';')
+                .withIgnoreQuotations(true)
+                .build();
+
+        try (CSVReader csvReader = new CSVReaderBuilder(new FileReader(filePath))
+                .withCSVParser(parser)
+                .build();
+        ) {
+            return csvReader.readAll()
+                    .stream()
+                    .sorted(new RowComparator())
+                    .collect(Collectors.toList());
+        }
+        catch(Exception e) {
+            throw new RuntimeException(
+                    "TestRequirementProcessor error: Can't read this file: " + filePath);
+        }
+    }
+
+    private class RowComparator implements Comparator<String[]> {
+        // Order rows of csv by TEST METHOD, REQUIREMENT
+        public int compare(String[] a, String[] b)
+        {
```

Comment:
Please move the opening bracket to the end of the previous line

Reply:
OK